packages/flutter/lib/src/rendering/view.dart

```diff
@@ -247,6 +247,12 @@ class RenderView extends RenderObject with RenderObjectWithChildMixin<RenderBox>
     }
   }
 
+  /// Sends a [SemanticsUpdate] to the semantics tree contained within the
+  /// window of this [RenderView].
```

Comment:
Maybe: 

Sends the provided [SemanticsUpdate] to the [FlutterView] associated with this [RenderView].

A [SemanticsUpdate] is produced by a [SemanticsOwner] during the [EnginePhase.flushSemantics] phase.